src/desktop-lite/install.sh

```diff
@@ -299,9 +299,9 @@ startInBackgroundIfNotRunning()
 {
     log "Starting \$1."
```

Comment:
Can we bump the patch version in here? thanks! 

https://github.com/devcontainers/features/blob/883cf50eb84f2ad3dada748401a5ad6c06bf43b6/src/desktop-lite/devcontainer-feature.json#L3